releasenotes/notes/enable-operation-name-v2-by-default-otel-agent-e62cc29df3658bdf.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Please remove the changelog entry. We do not want these changes reflected in the chagelog at this time.